app/models/admin.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Admin < ApplicationRecord
```

Comment:
🤔 As discussed in person, it's a common pattern to have an `admin` flag on a `user` model than a separate `admin` model, but there could be some legitimate reasons to have this separate..

Reply:
Thanks for flagging this, I didn't think about it very deeply and it's good to be reminded it's not a common pattern. I'm trying to remember where I got this from - I definitely used it ~2013 in the UK on enterprisey .Net systems which is rather a different world. 

I chose it here because in this case Users (public/caregivers/parents) are entirely different from Admins (probably just us developers tinkering with the data) and ELSs (kindergartens and stuff who will receive shared data) and will never use the same pages. I expect it to be easy to create e.g. Admin-only controllers by inheriting from `Admin::BaseController` which enforces admin authentication.

Reply:
Do you expect ELSs to be a type of User?

Reply:
I expect them to be a third authenticatable model, which doesn't overlap with Users or Admins. Not super clear on what they need to do yet but I expect them to be just the same as the setup for Users and Admins here.

Reply:
i'm really not sure about the 2 models. I know you can have poly model stuff. Jimsy from Rabid was super keen - but i don't see the advantage for this app.

Reply:
Well, doing it this way we're basically done with authentication. Don't need to implement roles or anything.